core/src/main/java/org/bitcoinj/protocols/channels/PaymentChannelClient.java

```diff
@@ -369,9 +369,9 @@ private void receiveRefund(Protos.TwoWayChannelMessage refundMsg, @Nullable KeyP
             // Make an initial payment of the dust limit, and put it into the message as well. The size of the
             // server-requested dust limit was already sanity checked by this point.
             PaymentChannelClientState.IncrementedPayment payment = state().incrementPaymentBy(Coin.valueOf(minPayment), userKey);
-            Protos.UpdatePayment.Builder initialMsg = contractMsg.getInitialPaymentBuilder();
-            initialMsg.setSignature(ByteString.copyFrom(payment.signature.encodeToBitcoin()));
-            initialMsg.setClientChangeValue(state.getValueRefunded().value);
+//            Protos.UpdatePayment.Builder initialMsg = contractMsg.getInitialPaymentBuilder();
+//            initialMsg.setSignature(ByteString.copyFrom(payment.signature.encodeToBitcoin()));
+//            initialMsg.setClientChangeValue(state.getValueRefunded().value);
```

Comment:
I will investigate the reason for these lines being commented as well.

This is not part of the bitcoinj PR that was used as a reference.

Reply:
This section of code and many others related to payment channels was removed from bitcoinj on July 4, 2019.

https://github.com/schildbach/bitcoinj/commit/d67f060e27f38e4117ba20f38d6ae8bd1130ab34

Is any of this related to BIP70?

Reply:
`Protos.UpdatePayment.Builder initialMsg = provideContractBuilder.getInitialPayment().toBuilder();`

The first line of the three can be changed to this.